### PR TITLE
Create config for FPR.2 and FPR.4 testcases

### PR DESCRIPTION
**Pull request by gilschieber**
Noticed that we use 2 peer SASs in these configs, but they use same certs.
This is also the case for similar testcases EPR, GPR, PPR.

Will address this issue in a separate PR.

### CODE REVIEW

File: src/harness/testcases/WINNF_FT_S_FPR_testcase.py [hunk 1]
```diff
@@ -28,6 +30,483 @@ def setUp(self):
...
+    fss_data['ttc'] = True
+
+    # Load devices and grant info for Scenario 1.
+    # Loading device_1 (Cat A) to a location within 40 km of FSS,
+    # frequency range outside of FSS band.
+    device_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_1['installationParam'][
+        'latitude'] = fss_record['installationParam']['latitude'] + 0.20
+    device_1['installationParam'][
```
Comment (jcereced) [inline src/harness/testcases/WINNF_FT_S_FPR_testcase.py]: I'm getting a failure when trying to generate the configs.  I think you need to add 

['deploymentParam'][0]

when you are modifying the lat/longs of the cbsd based on the lat/longs of the fss..

device_1['installationParam'][
+        'latitude'] = fss_record['installationParam']['latitude'] + 0.20

between fss_record and ['installationParam']

Comment (gilschieber) [reply src/harness/testcases/WINNF_FT_S_FPR_testcase.py]: Oups, sorry about that. Corrected.

File: src/harness/testcases/WINNF_FT_S_FPR_testcase.py [hunk 2]
```diff
@@ -28,6 +30,483 @@ def setUp(self):
...
+    }
+    cbsd_records_domain_proxy_1 = {
+        'registrationRequests': [device_3, device_4, device_7],
+        'grantRequests': [grant_request_3, grant_request_4, grant_request_7]
+    }
+
+    # Protected entity record
+    protected_entities = {
+        'fssRecords': [fss_record]
+    }
```
[the remark this answers is not in the document]
Comment (gilschieber) [reply src/harness/testcases/WINNF_FT_S_FPR_testcase.py]: You're right. Thanks for catching this. Done.

**Comment by gilschieber** [comment]
Thanks for the review.

**Pull request merged**